pxtlib/util.ts

```diff
@@ -1187,22 +1205,21 @@ namespace ts.pxtc.Util {
         }
 
         pxt.debug(`loc: ${code}`);
+
+        const liveUpdateStrings = pxt.Util.liveLocalizationEnabled()
         return downloadTranslationsAsync(targetId, baseUrl, code,
-            pxtBranch, targetBranch, live,
+            pxtBranch, targetBranch, liveUpdateStrings,
             ts.pxtc.Util.TranslationsKind.Editor)
             .then((translations) => {
                 if (translations) {
                     setUserLanguage(code);
                     setLocalizedStrings(translations);
-                    if (live) {
-                        localizeLive = true;
-                    }
                 }
 
                 // Download api translations
-                return !live ? ts.pxtc.Util.downloadTranslationsAsync(
+                return liveUpdateStrings ? ts.pxtc.Util.downloadTranslationsAsync(
```

Comment:
This is reversing the behavior but it definitely *looks* like it was incorrect before to me / not triggering correctly -- before it was saying 'if we're not doing live translation updates, try and download translations for apis'.